acceptance/setup/aio/pre-suite/021_InstallAristaModule.rb

```diff
@@ -12,6 +12,8 @@
 
   step 'add puppet user to switch' do
     switchs.each do |switch|
+      on(switch, "useradd -U puppet")
```

Comment:
@joshcooper & I chatted. Arista will have to inform customers to add puppet user and group.  I will remove the 'missinguser' and add confines to the other tests